.github/workflows/build-and-deploy.yml

```diff
@@ -125,6 +133,15 @@ jobs:
         run: |
           cp -avr architectures/target/generated-docs/ target/generated-docs/website/pages/architectures/
 
+      - name: Create output for tutorials
+        run: |
+          chmod +x wiki-tutorials/scripts/createOutput.sh
+          wiki-tutorials/scripts/createOutput.sh
+
+      - name: Copy output of tutorials
+        run: |
+          cp -avr wiki-tutorials/target/generated-docs/ target/generated-docs/website/pages/tutorials/
```

Comment:
Please copy everything to pages/learning/. Otherwise we will get strange URLs for the wiki tutorials later.